Changes

```diff
@@ -73,6 +73,10 @@ Working version
   stubs.
   (Mark Shinwell, review by Pierre Chambart and Leo White)
 
+- GPR#1195: Merge functions based on partiality rather than
+  Parmatch.irrefutable.
+  (Leo White, revuew by Thomas Refis and Gabriel Scherer)
```

Comment:
review

Reply:
While we are at it, you should add Alain in the review list, even if his first round of review was a bit too enthusiastic.